service/src/main/java/uk/gov/hmcts/reform/fpl/model/MigratedChildren.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class MigratedChildren {
```

Comment:
Rename to `Child`. Avoid using migrated word unless strictly needed.

Reply:
Children you mean of course in this case.

Reply:
What do you think about changing ChildParty to Child while i'm making the above change?